File: testdata/dirty/sample.zig
const std = @import("std");

// BARK: Remove debug prints before release
pub fn main() !void {
    std.debug.print("Debug mode\n", .{});
    
    // BARK: Replace with proper error handling
    const result = calculate(5, 3);
    std.debug.print("{}\n", .{result});
}

// Regular comment
fn calculate(a: i32, b: i32) i32 {
    /// BARK: This is a temporary implementation
    return a + b;
}

// BARK needs optimization
fn processData() void {
    // Implementation
}

File: testdata/dirty/sample.rs
// BARK: Remove debug prints before release
fn main() {
    println!("Debug mode");
    
    // BARK: Replace with proper error handling
    let result = calculate(5, 3);
    println!("{}", result);
}

// Regular comment
fn calculate(a: i32, b: i32) -> i32 {
    /* BARK: This is a temporary implementation
       Need to add proper validation */
    a + b
}

// BARK needs optimization
fn process_data() {
    // Implementation
}

